### ISSUE TITLE
Security HUDs broken by Z-levels

### ISSUE THREAD

**Issue by Ekaterina-von-Russland** [tgstation/tgstation#73134]
## Testmerges:

Doesn't seem like there were any 

## Reproduction:

When moving from one Z-level to the other Z-level, security HUDs broke. Mindshield outlines wouldn't show up at all - existing job icons would remain but new players entering my FOV would not have a job icon. I had to remove my HUDs and put them back on to fix this.

**Comment by Ekaterina-von-Russland** [comment]
Could maybe be related to https://github.com/tgstation/tgstation/issues/73011

**Comment by krimmity** [comment]
this happens with any huds, for some reason new player mobs don't get their medhud/sechud icon updated (WEIRD GHOST STUFF O_O)

**Comment by Jackraxxus** [comment]
This affects team antag (like revs) HUDs as well q.q

**Comment by Ekaterina-von-Russland** [comment]
Seems to be the same issue as https://github.com/tgstation/tgstation/issues/73075

**Comment by DaydreamIQ** [comment]
This ALSO affects medhuds so it probably affects all kinds

**Comment by TheSmallBlue** [comment]
Affects cyborg HUDS too so yeah this is global

**Comment by JupiterJaeden** [comment]
I don't believe this is an issue anymore, I do remember it but we haven't been having this problem for a long time. I'm not sure exactly which PR fixed it but, it's fixed.
